tutorial/practicals/generic/1_intro_psyir/README.md

```diff
@@ -10,7 +10,7 @@ this mini-app was kindly provided by Silvia Mocavero of CMCC.
 ## Prerequisites ##
 
 The requirements for this section are as described in the practicals
-[README.md](../../README.md#Requirements).
+[README.md](../../README.md##Requirements).
```

Comment:
Presumably the "##" is required?

Reply:
Yes, linkspector fails with the single hash saying that the anchorage point does not exist.